app/views/shared/_order_detail_cell.html.haml

```diff
@@ -6,6 +6,10 @@
     - if show_reservation && od.reservation
       %br
       %em= od.reservation
+
+    - if od.product.offline?
```

Comment:
I was thinking we were talking about the reservations tab while managing a facility. Under "My Reservations" we definitely need some sort of warning/icon there. In this view, if it's hard to only show it in one of the views, then I think I would lean towards the potential confusion if that means the user will see a notice where they need it.